x-pack/plugins/data_quality/common/url_schema/dataset_quality_url_schema_v1.ts

```diff
@@ -54,7 +24,6 @@ export const urlSchemaRT = rt.exact(
   rt.partial({
     v: rt.literal(1),
     table: tableRT,
-    flyout: flyoutRT,
```

Comment:
What should we do with this piece? What if a user saved an old URL with the flyout information?
Should we handle this in some way?

Reply:
I thought about it. The version bump would have made sense, if we would have done some parameters change as that's why the URL schema validation takes care of. But here we have literally introduced a new page at `/details` and hence i didn't do it.

Also looking at Telemetry, Dataset Quality is not extensively used at the moment and also its in BETA so i would avoid the unnecessary complexity here